lib/sandbox/pmapi.js

```diff
@@ -104,6 +106,9 @@ Postman = function Postman (bridge, execution, onRequest) {
         cookies: execution.cookies
     });
 
+    // Making a copy of the current context.
+    self = this;
```

Comment:
This assignment can be shifted above to the declaration.

Reply:
👍